staging/src/k8s.io/apiserver/pkg/server/routes/openapi.go

```diff
@@ -139,8 +140,8 @@ func (p *openAPIServiceProvider) Register() {
 	}
 
 	err = defaultOpenAPIService.RegisterOpenAPIVersionedService(p.path, &clusterAwarePathHandler{
-		clusterName: "",
-		addHandlerForCluster: func(clusterName string, handler http.Handler) {
+		clusterName: logicalcluster.LogicalCluster{},
```

Comment:
(No action required) Maybe standardize on a preferred empty form (e.g. `New("")` or `LogicalCluster{}` or even `NewEmpty()`)?

Reply:
interestingly you cannot use LogicalCluster{} everywhere